src/ProjectTemplates/ComponentsWebAssembly.ProjectTemplates/content/ComponentsWebAssembly-CSharp/Client/Program.cs

```diff
@@ -40,22 +40,22 @@ public static async Task Main(string[] args)
     #else
             builder.Services.AddOidcAuthentication(options =>
             {
-                #if(MissingAuthority)
+                #if (MissingAuthority)
                 // Configure your authentication provider options here.
                 // For more information, see https://aka.ms/blazor-standalone-auth
                 #endif
-                options.ProviderOptions.Authority = "https://login.microsoftonline.com/";
-                options.ProviderOptions.ClientId = "33333333-3333-3333-33333333333333333";
+                builder.Configuration.Bind("Authority", options.ProviderOptions.Authority);
```

Comment:
We probably want to do this at the top level? The goal here is that customers can bind an json payload with the same shape as the object. Like builder.Configuration.Bind("AzureAd", options) and so on.